test/e2e/systemd_test.go

```diff
@@ -176,4 +176,21 @@ WantedBy=multi-user.target
 
 		Expect(session.OutputToString()).To(Not(ContainSubstring("noexec")))
 	})
+
+	It("podman run --systemd arg is case insensitive", func() {
+		session := podmanTest.Podman([]string{"run", "--rm", "--systemd", "Always", ALPINE, "echo", "test"})
```

Comment:
Could we use `create` instead of `run` to spare some CI cycles?

Reply:
No it failed at runtime before